pkg/drivemgr/loopbackmgr/Dockerfile.build

```diff
@@ -1,5 +1,7 @@
 FROM    ubuntu:21.04
 
+RUN     sed -i -re 's/archive.ubuntu.com|security.ubuntu.com/old-releases.ubuntu.com/g' /etc/apt/sources.list \
```

Comment:
Can you use `add-apt-repository` tool instead?

Reply:
No, base image doesn't contain `add-apt-repository`